spec/all_approval_git_reflow_spec.rb

```diff
@@ -18,6 +18,11 @@
   let(:existing_pull_request)  { GitReflow::GitServer::GitHub::PullRequest.new existing_pull_requests.first }
 
   before do
+
```

Comment:
this file has still been renamed.  Let's keep the file naming as it was and just keep the default settings tested here (as it is with all approvals).  `git mv spec/all_approval_git_reflow_spec.rb spec/git_reflow_spec.rb` should do it